docs/se/webclient.adoc

```diff
@@ -157,94 +303,87 @@ client:
+  tls: <7>
+    trust:
       keystore:
         passphrase: "password"
         trust-store: true
         resource:
           resource-path: "client.p12"
-    client:
-      keystore:
-        passphrase: "password"
-        resource:
-          resource-path: "client.p12"
 ----
 
 <1> Client functional settings
-<2> Default client headers and cookies
-<3> Client service configuration
-<4> Proxy configuration
-<5> TLS configuration
+<2> Cookie management
+<3> Default client headers
+<4> Client service configuration
+<5> Protocol configuration
+<6> Proxy configuration
+<7> TLS configuration
 
 == Examples
 
-=== Request Configuration
+=== Webclient with Proxy
+Configure Proxy setup either programmatically or via the Helidon configuration framework.
 
-The request settings are based on the following optional parameters, and change when a specific request is executed.
-
-|===
-|Parameter |Description
-
-|`uri("http://example.com")`   |Overrides baseUri from WebClient
-|`path("/path")`   |Adds path to the uri
-|`queryParam("query", "parameter")`   |Adds query parameter to the request
-|`fragment("someFragment")`   |Adds fragment to the request
-|`headers(headers -> headers.addAccept(MediaType.APPLICATION_JSON))`   |Adds header to the request
-|===
-
-`WebClientRequestBuilder` class also provides specific header methods that help the user to set a particular header. The methods are:
+==== Configuring Proxy in your code
+Proxy can be set directly from WebClient builder.
+[source,java]
+----
+Proxy proxy = Proxy.builder()
+                .type(Proxy.ProxyType.HTTP)
+                .host(PROXY_HOST)
+                .port(PROXY_PORT)
+                .build();
+WebClient webClient = WebClient.builder()
+         .proxy(proxy)
+         .build();
+----
 
-* `contentType` (MediaType contentType)
-* `accept` (MediaType... mediaTypes)
+Alternative is to set proxy directly from the request via  `HttpClientRequest`.
 
-For more details, see the link:{webserver-javadoc-base-url}/io/helidon/webserver/RequestHeaders.html[Request Headers] API.
+[source,java]
+----
+// Using System Proxy
+Proxy proxy = Proxy.create();
+System.setProperty("http.proxyHost", PROXY_HOST);
+System.setProperty("http.proxyPort", PROXY_PORT);
+System.setProperty("http.nonProxyHosts", "localhost|127.0.0.1|10.*.*.*|*.example.com|etc|" + TARGET_HOST);
+webClient.get("/proxiedresource").proxy(proxy).request()
+----
 
-=== Adding JSON Processing Media Support to the WebClient
+==== Configuring Proxy in the config file
 
-JSON Processing (JSON-P) media support is not present in the WebClient by default. So, in this case, you must first register it before making a request.
-This example shows how to register `JsonpSupport` using the following two methods.
+Proxy can also be configured in WebClient through the `application.yaml` configuration file.
 
-[source,java]
-.Register JSON-P support to the WebClient.
+[source,yaml]
+.WebClient TLS configuration file `application.yaml`
```

Comment:
I think you overlooked here. It should be Proxy instead of TLS

Reply:
Corrected and did the same on the `Service` section

Reply:
resolved